.azuredevops/modulePipelines/ms.authorization.policysetdefinitions.yml

```diff
@@ -22,6 +22,7 @@ trigger:
       - '/.azuredevops/modulePipelines/ms.authorization.policysetdefinitions.yml'
       - '/.azuredevops/pipelineTemplates/module.*.yml'
       - '/arm/Microsoft.Authorization/policySetDefinitions/*'
+      - '/arm/.global/**'
```

Comment:
everything in global? I'm just wondering as we have also dependency templates in that folder (in the dependency PoC). And not every module will use them - but would be triggered regardless

Reply:
True, although I now wonder if anything not having a global impact should be included in a folder named global

Reply:
A good point. One could however, also argue that `global` = `shared` and not every module has to consume everything that is shared. This is some deep philosophical conversation right here

Reply:
I'm not even sure why the .global folder is in the arm folder actually. I think this should be discussed as well. Isn't it a utility pipeline afterall?

Reply:
It is there as it is module specific. In any case I'd suggest to open this as a new discussion and keep separate from this PR. For the time being I'd anyhow strongly suggest to be more specific than `global*`

Reply:
This would be listing all files contained in the .global folder right now. Fine but hopefully temporary 😆

Reply:
Aren't we after the `global.tests.ps1`? :D

Reply:
and related helper. current and future.